### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by faroit** [mpariente/asteroid#53]
Do you plan to add support for pretrained models (maybe through torchhub?) I think that would make a really nice addition. 

E.g. recipes could be different for training and inference.

**Comment by mpariente** [comment]
> Do you plan to add support for pretrained models (maybe through torchhub?) I think that would make a really nice addition.

Yes, this is on the list of things we'd like to do. I haven't looked into it yet, but yes definitely. 
> 
> E.g. recipes could be different for training and inference.

I don't understand what you mean. Can you explain please?

**Comment by faroit** [comment]
> >  E.g. recipes could be different for training and inference.

> I don't understand what you mean. Can you explain please?

you could maybe add recipes that only serve the purpose of using pretrained models. So that they could easily be deployed on cloud platforms for example

**Comment by mpariente** [comment]
Some news about this. 
Models are derived from data. So models trained on data with restrictive licenses cannot be released. 
wsj0-mix, WHAM and WHAMR are based on WSJ0 which is not open, hence models cannot be released..

That would be another advantage of LibriMix.

**Comment by faroit** [comment]
>  Models are derived from data. So models trained on data with restrictive licenses cannot be released.

I thought this is going to be changed in the EU and is already in place in France. Maybe good to ask inria administration to get the current status on this.

**Comment by mpariente** [comment]
Do you have any resources regarding this change? A link to an article or anything?

**Comment by mpariente** [comment]
Alright, we have the permission of LDC to share wsj0-based models, the first one has been shared. 
I think this can be closed !
